src/client/app/core/datepicker/datepicker.js

```diff
@@ -0,0 +1,78 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.core')
+        .controller('datePickerCtrl', datePickerCtrl)
+
+    function datePickerCtrl() {
```

Comment:
though this one is not too important, I'd recommend changing this to DatePickerController as well.